rust/template/ddlog_derive/tests/ui/pass/mutator.rs

```diff
@@ -32,3 +36,14 @@ impl Default for Enum {
         Self::Baz
     }
 }
+
+#[derive(FromRecord, IntoRecord, Mutator, Debug, Clone, PartialEq, Deserialize)]
+struct NestedMaps {
+    m: BTreeMap<BTreeMap<u64, String>, BTreeMap<String, BTreeMap<u32, u32>>>
+}
+
+#[derive(FromRecord, IntoRecord, Mutator, Debug, Clone, PartialEq, Deserialize)]
+enum ContainersEnum {
+    Map(BTreeMap<Enum, Struct>),
+    Vec(Vec<NestedMaps>)
+}
```

Comment:
You can run actual tests within the compile tests by writing code in the `main()` function, I'd move this to another file and write some expected behavior assertions in the main function

Reply:
Done, thanks!